neo/SmartContract/ContractManifest.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Neo.Ledger;
+using System.Linq;
+
+namespace Neo.SmartContract
+{
+    /// <summary>
+    /// When a smart contract is deployed, it must explicitly declare the features and permissions it will use.
+    /// When it is running, it will be limited by its declared list of features and permissions, and cannot make any behavior beyond the scope of the list.
+    /// </summary>
+    public class ContractManifest
+    {
+        /// <summary>
+        /// Contract hash
+        /// </summary>
+        public UInt160 Hash { get; set; }
+
+        /// <summary>
+        /// A group represents a set of mutually trusted contracts. A contract will trust and allow any contract in the same group to invoke it, and the user interface will not give any warnings.
+        /// The group field can be null.
+        /// </summary>
+        public ContractManifestGroup Group { get; set; }
```

Comment:
It can just be a part of a single Group?

Reply:
I think that it could be an array.